onnx/mapping.py

```diff
@@ -19,6 +19,7 @@
     int(TensorProto.INT64): np.dtype('int64'),
     int(TensorProto.BOOL): np.dtype('bool'),
     int(TensorProto.FLOAT16): np.dtype('float16'),
+    int(TensorProto.BFLOAT16): np.dtype('float16'),
```

Comment:
add a comment explaining why you are mapping bfloat16 to np float16 something like (numpy does not support bfloat16...)